src/go/rpk/pkg/tuners/iotune/data.go

```diff
@@ -250,6 +250,50 @@ func precompiledData() map[string]map[string]map[string]io {
 				// advertised:      3200000                       1760000
 				"default": {"", 16 * 200633, 16 * 1427463808, 16 * 110542, 16 * 1113520256},
 			},
+
+			// m7gd values are taken from direct measurement with iotune except
+			// where noted.
+			"m7gd.large": {
+				// advertised:  33542             16771
+				"default": {"", 33638, 157521648, 16836, 75264032},
+			},
+
+			"m7gd.xlarge": {
+				// advertised:  67084             33542
+				"default": {"", 67274, 315774176, 33676, 150679296},
+			},
+
+			"m7gd.2xlarge": {
+				// advertised:  134168             67084
+				"default": {"", 134550, 631589440, 67358, 301293568},
+			},
+
+			"m7gd.4xlarge": {
+				// advertised:  268336              134168
+				"default": {"", 269588, 1263357056, 134608, 604995968},
+			},
+
+			// at 8xlarge and above the read-side IOPS values start to scale
+			// more poorly that advertised/expected, so use scaled values
+			// from 4xlarge instead, under the assumption it is a measurement
+			// limitation
```

Comment:
I believed that when I wrote it but after looking at it more, I believe it is probably not a measurement limitation but +1 to this for consistency anyway.